src/main/java/com/empowerhack/family/memberservice/Entity/Skill.java

```diff
@@ -12,5 +14,14 @@
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
 
+    @NotNull
+    @Size(min=2, max=32)
     private String name;
+
+    Skill() {
```

Comment:
Surely not having visibility on the constructor would make it private though..   Weird